doc/btp_gap.txt

```diff
@@ -695,6 +695,88 @@ Commands and responses:
+                Command parameters:     Address_Type (1 octet)
+                                        Address (6 octets)
+                                        SID (1 octet)
+                                        Num_BIS (1 octet)
+                                        BIS_Bitfield (4 octets)
+                                        MSE (4 octets)
+                                        Sync_Timeout (2 octets)
+                                        Encryption (1 octet)
+                                        Broadcast_Code (0 octet or 16 octets)
+                Return Parameters:      <none>
+
+                This command is used to synchronize to a BIG. Before executing this command, the
+                IUT should be synced to a PADV. It means the `Periodic Sync Established event`
+                (opcode 0x8d) should be received before executing this command.
+
+                Possible values for the BIS_Bitfield parameter are a bit-wise OR of the bits 0-31.
+                Such as, bit 0 represents BIS index 1, bit 1 represents BIS index 2, and so on.
+                And the number of bits set by the BIS_Bitfield parameter should be consistent with
+                the Num_BIS parameter.
+
+                Available Encryption:
+                0 - The BIS will be unencrypted. The Broadcast_Code parameter should be omitted.
+                1 - The BIS will be encrypted. The Broadcast_Code parameter should be existing and
+                    the length should be 16 octets.
+
+                In case of an error, the error response will be returned.
+                In case of a success, the IUT continues processing the command asynchronously.
+
+                After the command is processed, the IUT shall setup the Data Path for all the BIS,
+                and generate a "BIS Data Path Setup Event" for each BIS.
+
+        Opcode 0x2c - Create BIG
+
+                Controller Index:       <controller id>
+                Command parameters:     ID (1 octet)
+                                        Num_BIS (1 octet)
+                                        Interval (4 octets)
+                                        Latency (2 octets)
+                                        RTN (1 octet)
+                                        PHY (1 octet)
+                                        Packing (1 octet)
+                                        Framing (1 octet)
+                                        Encryption (1 octet)
+                                        Broadcast_Code (0 octet or 16 octets)
+                Return Parameters:      <none>
+
+                This command is used to create a BIG. The parameter ID is corresponding to a
+                specific periodic advertising.
+
+                Possible values for the PHY parameter are a bit-wise or of the following bits:
+                        0 = PHY 1M
+                        1 = PHY 2M
+                        2 = PHY Coded
```

Comment:
```suggestion
                Possible values for the PHY parameter are a bitfield or of the following bits:
                        0 = PHY 1M
                        1 = PHY 2M
                        2 = PHY Coded
```

What does it mean if you provide e.g. `0b111`? Why can more than 1 bit be set?

Reply:
Right, but I think that @sjanc 's plan is to not lean too much on the HCI spec, so we need to specify this in a BTP context